compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__wrap_pipeline_step_in_anonymous_function.snap

```diff
@@ -0,0 +1,32 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "pub fn main() {\n  1 |> wibble |> wobble\n}\n\nfn wibble(i) {\n  todo\n}\n\nfn wobble(i) {\n  todo\n}\n\n"
+---
+----- BEFORE ACTION
+pub fn main() {
+  1 |> wibble |> wobble
```

Comment:
Could we have a test for wibble taking 2 arguments please

Reply:
added